nautobot/circuits/navigation.py

```diff
@@ -0,0 +1,105 @@
+from nautobot.extras.nautobot_app import NavMenuButton, NavMenuItem, NavMenuTab, NavMenuGroup
+from nautobot.utilities.choices import ButtonColorChoices
+
+
+menu_tabs = (
+    NavMenuTab(
+        name="Circuits",
+        weight=500,
+        groups=(
+            NavMenuGroup(
+                name="Circuits",
+                weight=100,
+                items=(
+                    NavMenuItem(
+                        link="circuits:circuit_list",
+                        link_text="Circuits",
+                        permissions=[
+                            "circuits.view_circuit",
+                        ],
+                        buttons=(
+                            NavMenuButton(
+                                link="circuits:circuit_add",
+                                title="Circuits",
+                                icon_class="mdi mdi-plus-thick",
+                                color=ButtonColorChoices.GREEN,
+                                permissions=[
+                                    "circuits.add_circuit",
+                                ],
+                            ),
+                            NavMenuButton(
+                                link="circuits:circuit_import",
+                                title="Circuits",
+                                icon_class="mdi mdi-database-import-outline",
+                                color=ButtonColorChoices.BLUE,
```

Comment:
Instead of directly specifying the color (I know this is inherited from the Plugin API) it would be more self-consistent to have a `button_class` property that applies a CSS class to the `button` element instead. This would also allow for the buttons to retain their existing styling (`btn-success` / `btn-info`) instead of (BLUE / GREEN, which are similar but *not exactly* the same colors).